docs/current/guides/snippets/azure-pipelines-container-instances/main.py

```diff
@@ -0,0 +1,97 @@
+import sys
+import os
+
+import anyio
+import dagger
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.containerinstance import ContainerInstanceManagementClient
```

Comment:
Should use the async client:
```suggestion
from azure.mgmt.containerinstance.aio import ContainerInstanceManagementClient
```